docs/apache-airflow/lineage.rst

```diff
@@ -71,7 +71,7 @@ for the downstream task.
 
 .. note:: Operators can add inlets and outlets automatically if the operator supports it.
 
-In the example DAG task ``run_this`` (task_id=``run_me_first``) is a BashOperator that takes 3 inlets: ``CAT1``, ``CAT2``, ``CAT3``, that are
+In the example DAG task ``run_this`` (task_id = ``run_me_first``) is a BashOperator that takes 3 inlets: ``CAT1``, ``CAT2``, ``CAT3``, that are
```

Comment:
```suggestion
In the example DAG task ``run_this`` (``task_id=run_me_first``) is a BashOperator that takes 3 inlets: ``CAT1``, ``CAT2``, ``CAT3``, that are
```

Whitespace is not the problem, this is formatted incorrectly.

Reply:
Thank you for telling the original intention :) Adjusted accordingly. Please reconfirm.